NetCasbin/Model/Policy.cs

```diff
@@ -91,29 +93,71 @@ public List<List<string>> GetFilteredPolicy(string sec, string ptype, int fieldI
 
         public bool HasPolicy(string sec, string ptype, List<string> rule)
         {
-            return Model[sec][ptype] != null && Model[sec][ptype].Contains(rule);
+            var assertion = GetExistAssertion(sec, ptype);
+            return assertion.Contains(rule);
+        }
+
+        public bool HasPolicies(string sec, string ptype, IEnumerable<List<string>> rules)
```

Comment:
This method is not test-coveraged, would be great to add some.

Reply:
Ok

Reply:
The `HasPolicies` method will be called at InternalAdd/RemovePolicies and the test has call those apis by e.Add/RemovePolicies.  The go casbin looks like also has not more test case.
I will add more tests later at the pull request for issue #76 .